corporate/lib/analytics.py

```diff
@@ -71,6 +71,8 @@ def get_plan_data_by_remote_server() -> Dict[int, RemoteActivityPlanData]:  # no
             renewal_cents = RemoteRealmBillingSession(
                 remote_realm=plan.customer.remote_realm
             ).get_customer_plan_renewal_amount(plan, timezone_now())
+            if plan.customer.remote_realm.registration_deactivated:
+                continue
```

Comment:
Hmm, what are the invariants we have about this? I think the intent would be that one can't deactivate the registration for a realm without first cancelling any plans for it -- I worry a bit that this might paper over realms that escape that invariant? But maybe we should just do a pass of making such outcomes not possible.

Though maybe this isn't the right place to handle such invariant violations.